### DISCUSSION TITLE
Enhance version check to differentiate between ESP32 and ESP32-S3 bin versions

### DISCUSSION THREAD

**Discussion by tp1de** [emsesp/EMS-ESP32#1238] [Ideas]
The System / Version Check from web-ui is a nice function. 
I am currently using the S3 version. The download functionality does not  differentiate between the esp32 and the esp32-s3 bin file. 

Since @bbqkees is working on a S3 version of his gateway, it might be a good idea to download the correct bin file from web-ui. 
Version 3.6. should be productive soon. I would recommend to add this functionality. (3.6 might stay for a couple of months)

@MichaelDvP @proddy  - What's your opinion?

**Comment by proddy** [comment]
Yes, this would be a good enhancement and easy to implement. I’ll do a PR
